js/renovation/ui/grids/data_grid_next/scrolling/scrolling.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Component, JSXComponent, ComponentBindings,
+  OneWay, Fragment,
+} from '@devextreme-generator/declarations';
+import { ValueSetter } from '../../../../utils/plugin/value_setter';
+import { VirtualScrolling } from './virtual_scrolling';
+import { ScrollingMode, VirtualScrollingMode } from '../types';
+import { ScrollingModeValue } from './plugins';
+
+export const viewFunction = ({
+  isVirtualScrolling, virtualScrollingMode,
+  props: {
+    mode,
+  },
+}: DataGridNextScrolling): JSX.Element => (
+  <Fragment>
+    <ValueSetter type={ScrollingModeValue} value={mode} />
+    {isVirtualScrolling && (<VirtualScrolling mode={virtualScrollingMode} />)}
```

Comment:
```suggestion
    {mode !== 'standard' && (<VirtualScrolling mode={mode} />)}
```

In this case there is no need in confusing getter `virtualScrollingMode`